docs/concepts.md

```diff
@@ -32,15 +32,36 @@ Each Cloud Deployment must specify an environment name to deploy and a git branc
 >
 > All operations that perform compute require a deployed environment - including: ad-hoc job execution, EL pipelines, scheduled tasks, etc.
 
-## Meltano Cloud Runs
+## Meltano Cloud Jobs
 
-A run within Meltano is an instance of a Meltano operation that performed some compute. If you execute any of the following CLI commands, that is considered a run:
+A job within Meltano is an instance of a Meltano operation that performed some compute. If you execute any of the following CLI commands, that is considered a job:
 
 * invoke
 * elt
 * run
 * test
 
+Each job has additional adjectives that give more information about what kind of job it is. 
+
+### On-demand, Unnamed Jobs
+
+An unnamed job is one that is specified directly on the command line. If a user runs `invoke` or `test`, that is always an on-demand unnamed job. If a user runs `meltano run` or `meltano elt` and specifies each plugin on the command line, that is an unnamed job.
+
+### On-demand, Named Jobs
+
+A named job is defined in the meltano.yml file either under the `jobs` key in the case of `meltano run` or via the `schedules` key in the case of `meltano elt`. 
+
+On-demand, named jobs can be triggered via the following commands:
+
+* `meltano schedule run <schedule_name>` (this works for `meltano elt` or `meltano run`)
+* `meltano run <job_name>` (this works only for `meltano run`)
+
+### Scheduled Jobs
+
+Scheduled jobs are named jobs that have been set to run on an interval and have been triggered automatically according to that schedule. These are defined under the `schedules` key in the meltano.yml file. [Documentation here.](https://docs.meltano.com/concepts/project#schedules)
+
+All scheduled jobs are named jobs.
```

Comment:
@tayloramurphy - There's one case which I'm not sure is covered here, which is an EL workflow scheduled individually, and not via an explicitly created job.

EL tasks can still be scheduled and run without declaring in the named `jobs` list. In those cases, we could use the schedule name to refer to them, or just indicate that they are not technically named jobs even though they are scheduled.

I think I'm okay with any way you want to categorize those, but just wanted to call out this case.

Reply:
@tayloramurphy - Schedules have a name, but that is not the job name.

Taking the example (written by hand, so please pardon typos):

```yml
schedules:
- name: daily-prod
  job: full-build
  interval: @daily
- name: daily-jaffle-el
  extractor: tap-jaffle-shop
  loader: target-snowflake
  interval: @daily
```

Looking up executions of the `daily-prod` schedule will find executions with `schedule_name: daily-prod` and job_name: full-build`. Looking up executions of the `daily-jaffle-el` will find executions with `schedule_name: daily-jaffle-el` and `job_name: N/A`.

So there _is_ a name associated, but not a job name.

Reply:
@aaronsteers I think this points to us _not_ using `Jobs` as the identifier for operations in this case because it's already pretty confusing. In your example we're talking about Jobs at 2 different levels.

If we instead swap Jobs for Runs I think the language becomes cleaner. In your example an execution of `daily-jaffle-el` becomes a Named Run because it has a schedule name. We could infer a job_name for elt-based runs as well to make queries easier and prevent unexpected surprises when showing data. 

The questions folks might ask are:

* show me all Runs of Schedule `daily-prod`
* show me all Runs of Job `full-build`
* show me all Runs of Schedule `daily-jaffle-el`
* show me all Runs of Job (elt-tap-jaffle-shop-target-snowflake-skip or whatever we autogenerate - we wouldn't _recommend_ this method, but we could us it).

This points to a product improvement of collapsing the two schedule definitions into a single one like we've talked about previously.

So what do you think about updating the language to Runs?

Reply:
@aaronsteers I updated the language to be runs in this PR. let me know what you think. I personally like it.